build/Helix/runtests.cmd

```diff
@@ -8,10 +8,8 @@ reg add HKLM\Software\Policies\Microsoft\Windows\Appx /v AllowAllTrustedApps /t
 :: expected to show UI we don't want it running.
 taskkill -f -im dhandler.exe
 
-cd scripts
```

Comment:
I generally like to have scripts do "pushd %~dp0" just to guarantee that the working directory is stable for cases that you run runtests from a directory other than the one it's in. If you do it after "setlocal" then you don't even have to remember to "popd".

Reply:
I actually don't want to do that. The working directory on the Helix machine is NOT where the runtests.cmd script is. The first thing we do is to robobop everything into the working directory, but the script being executed is located elsewhere.

I think the idea is that '%HELIX_CORRELATION_PAYLOAD%' is an immutable set of files, and in theory multiple work items could execute on the same machine from the same source without having to redeploy the files. I can't say for certain whether the Helix implementation actually does that.

---
In reply to: [301832851](https://github.com/microsoft/microsoft-ui-xaml/pull/1025#discussion_r301832851) [](ancestors = 301832851)